packages/website/src/components/page-templates/package-home.js

```diff
@@ -89,10 +89,9 @@ const PackageHome = ({ data, children }) => {
             <Header id={data.id} heading={titleCase(data.packageName)} />
             <Description>{data.description}</Description>
             <PackageMetaData
+              fields={data.customPackageFields}
               id={data.id}
-              version={data.version}
-              maintainers={data.maintainers}
-              repository={data.repository}
+              metaData={data}
```

Comment:
Are both `fields` and `metaData` are necessary here? If the customPackageField values were added to its own property in `data`, e.g. `data.metaData`, rather than just spread into it, could we just pass in `metaData` without having to pass in a separate `data.customPackageFields` array?

Reply:
agreed there is a bit of duplication here. I think this is a larger task though as we should update all the references to these fields (i.e. the packages list page uses these fields)